requirements.txt

```diff
@@ -1,21 +1,12 @@
 aspy.yaml==1.3.0
-atomicwrites==1.3.0
-attrs==19.3.0
 cfgv==2.0.1
 identify==1.4.7
 importlib-metadata==0.23
 more-itertools==7.2.0
 nodeenv==1.3.3
-packaging==19.2
-paho-mqtt==1.4.0
```

Comment:
This is where the problem happens. You need to sync your `requirements.txt` with the one on master.